identity-aware-proxy/IAPClient/README.md

```diff
@@ -3,79 +3,23 @@
 This sample shows how to authenticate an http request when requesting a web page from
 a Google App Engine application protected by the [Google Cloud Identity-Aware Proxy](http://cloud.google.com/iap).
 
-This sample requires [.NET Core 2.0](
+This sample requires [.NET Core 3.1](

```

Comment:
Is there any reason for the odd line breaking here? It looks like it should be doable in just two lines.